src/pages/Overview/OverviewPage.tsx

```diff
@@ -620,6 +620,41 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
           namespaceActions.push(enableAction);
         }
       }
+      if (
+        serverConfig.kialiFeatureFlags.istioUpgradeAction &&
+        serverConfig.istioCanaryRevision.upgrade &&
+        serverConfig.istioCanaryRevision.current
+      ) {
+        const upgradeAction = {
+          isGroup: false,
+          isSeparator: false,
+          isDisabled: !canWrite,
+          title: 'Upgrade to ' + serverConfig.istioCanaryRevision.upgrade + ' Istio',
```

Comment:
As this action should be linked with [1] semantic, what do you think to use something like:

"Upgrade to XYZ revision"

That would help user to understand better the mapping between Kiali and this feature, as also users would need to deal with "tags" and also with other potential naming.

[1] https://istio.io/latest/docs/setup/upgrade/canary/

Reply:
agree